sdcm/nemesis.py

```diff
@@ -1362,8 +1362,6 @@ def disrupt_snapshot_operations(self):
         self._set_current_disruption('SnapshotOperations')
         result = self.target_node.run_nodetool('snapshot')
         self.log.debug(result)
-        if result.stderr:
-            self.tester.fail(result.stderr)
```

Comment:
please also fix:
- line 1365: ` result = self.target_node.run_nodetool('clearsnapshot')`
- replace calls to  self.tester.fail by throwing Exception
line 1371: `self.tester.fail('Snapshot %s creating failed %s' % (snapshot_name, result.stderr))`

Reply:
done